### PR TITLE
Nexmo SMS

### PR DESCRIPTION
**Pull request by PetrushynskyiOleksii**
## JIRA

* [PYTHON-217](https://ssu-jira.softserveinc.com/browse/PYTHON-217)

## Code reviewers

- [ ] @meelros
- [ ] @BoroviyOrest
- [ ] @Kardan1123
- [ ] @mizin4ik
- [ ] @dimigor 
- [ ] @Sasha1152

### Second Level Review

- [ ] @lhalam

## Summary of change
Implement possibility to send SMS.

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: way_to_home/utils/sms_helpers.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+"""This module provides helpers functions for sending SMS."""
+
+import nexmo
+
+from way_to_home.settings import NEXMO_API_KEY, NEXMO_API_SECRET
```
Comment (dimigor) [inline way_to_home/utils/sms_helpers.py]: `from django.conf import settings`
this is a better way to import settings

File: way_to_home/way_to_home/settings.py [hunk 2]
```diff
@@ -155,6 +155,12 @@
 JWT_TOKEN_KEY = 'any secret word'
 JWT_ALGORITHM = 'HS384'
 
+# Required setting for NEXMO
+
+NEXMO_API_KEY = 'Nexmo API KEY'
```
Comment (dimigor) [inline way_to_home/way_to_home/settings.py:164]: add these keys to local settings

Comment (PetrushynskyiOleksii) [reply way_to_home/way_to_home/settings.py:164]: This has already been added. The localsettings will overwrite these values.

File: way_to_home/utils/sms_helpers.py (hunk 1 above)
Comment (PetrushynskyiOleksii) [reply way_to_home/utils/sms_helpers.py]: Done, thank u

**Review by BoroviyOrest — DISMISSED** [review]

**Review by dimigor — DISMISSED** [review]

**Review by lhalam — DISMISSED** [review]

**Review by meelros — APPROVED** [review]
All is good!

**Review by BoroviyOrest — APPROVED** [review]

**Pull request merged**